config/puma_config.rb

```diff
@@ -8,7 +8,7 @@
 
 require 'bolt_server/transport_app'
 require 'bolt_server/acl'
-require 'bolt_server/config'
+require 'service_config/bolt_server_config'
```

Comment:
Looks like this should still be `bolt_server/config`?

Reply:
Also, this should have a more specific name than `puma_config.rb` now, since there's also a `plan_config.rb`. Maybe `transport_server_config.rb` or something? Given we have to make a corresponding bolt-vanagon change either way, this seems like the best time to rename it.